### DISCUSSION TITLE
Simpler way to create text annotations directly in PDF?

### DISCUSSION THREAD

**Discussion by praktikantonelli** [RyotaUshio/obsidian-pdf-plus#153] [Q&A]
Hi, I'm not sure if I just missed the intended way to do this, but I was wondering if it's possible to add a comment to a highlight in the PDF itself _aside from creating a highlight, then selecting it and clicking "Edit" and adding the text there_, What I would be looking for is something like a hotkey or a shortcut in the toolbar to highlight text and immediately open some sort of dialog to add a comment to the highlight.

**Comment by RyotaUshio** [comment]
Currently, PDF++ doesn't have such a shortcut. Since it is designed for adding comments in markdown files rather than in PDF itself, its support for this kind of feature is relatively scarce.
If this is your main use case, it might be better to use other apps like Adobe Acrobat.

**Reply by praktikantonelli** [reply]
alright, thanks for the answer. I'll see if I can get used to the intended use case
